hack/make/test-docker-py

```diff
@@ -55,8 +55,9 @@ PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_image_test.py
 			[ -n "${TESTDEBUG}" ] && set -x
 			[ -z "${TESTDEBUG}" ] && build_opts="--quiet"
 			[ -f /.dockerenv ] || build_opts="${build_opts} --network=host"
+			# TODO(thaJeztah): set a default VERSION in the tests/integration Dockerfile in docker-py or set SETUPTOOLS_SCM_PRETEND_VERSION conditionally
 			# shellcheck disable=SC2086
-			exec docker build ${build_opts} -t "${docker_py_image}" -f tests/Dockerfile "https://github.com/docker/docker-py.git#${DOCKER_PY_COMMIT}"
+			exec docker build ${build_opts} -t "${docker_py_image}" --build-arg VERSION="${DOCKER_PY_COMMIT}" -f tests/Dockerfile "https://github.com/docker/docker-py.git#${DOCKER_PY_COMMIT}"
```

Comment:
Oh no, yet another env var whose value then gets used for `SETUPTOOLS_SCM_PRETEND_VERSION` ? 🙈

Reply:
Yup; looks like it was there before, but at the time it had a default value in the Dockerfile (99.0.0?).

That default was removed, so now it must be passed. We should fix that, because I think we should try to make any Dockerfile have sensible defaults, so that they work without passing build-args (only in exceptional cases where a build-arg MUST be set).